src/libs/E2E/types.ts

```diff
@@ -39,6 +39,12 @@ type TestResult = {
     /** Optional, if set indicates that the test run failed and has no valid results. */
     error?: string;
 
+    /**
+     * Whether error is critical. If `true`, then server will be stopped and `e2e` tests will fail. Otherwise will simply log a warning.
+     * Default value is `true`
+     */
+    critical?: boolean;
```

Comment:
I believe our naming style guide in the App suggests to use:
```suggestion
    isCritical?: boolean;
```